README.md

```diff
@@ -83,6 +83,12 @@ If the error format does not suite your needs, you an encouraged to write your o
 
 `celebrate` exports the version of joi it is using internally. For maximum compatibility, you should use this version when passing in any validation schemas.
 
+### `isCelebrate(err)`
+
+Returns `true` if the provided `err` object originated from the `celebrate` middleware. Usseful if you want to write your own error handler for `celebrate` errors.
```

Comment:
Usseful -> Useful

Reply:
You should probably add ", and `false` otherwise" to the end of the first sentence.